Unable to build any rdkit >= 2025.3.6, when RDK_BUILD_DESCRIPTORS3D=OFF

**Describe the bug**

I am building an array of Rocky Linux-based postgres cartridge images and all the builds for  rdkit >= 2025.3.6 fail due to error in the GETAWAY tests

```sh
opt/RDKit-build/rdkit/Code/GraphMol/Descriptors/catch_tests.cpp:654:18: error: 'GETAWAY' is not a member of 'RDKit::Descriptors'
```

Looks like, definition of the GETAWAY has conditional logic depending on `RDK_BUILD_DESCRIPTORS3D`,  but the test does not, so the test fails when the flag is set to `NO`


**To Reproduce**
Build rdkit >= 2025.3.6 with `RDK_BUILD_DESCRIPTORS3D=OFF`

**Expected behavior**
Build is sucessful

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Configuration (please complete the following information):**
 - RDKit version: >=2025.3.6
 - OS: [e.g. Ubuntu 20.04] Rocky Linux 9
 - Python version (if relevant):
 - Are you using conda? No

--- a/Code/GraphMol/Descriptors/catch_tests.cpp
+++ b/Code/GraphMol/Descriptors/catch_tests.cpp
@@ -21,7 +21,9 @@
 #include <GraphMol/Descriptors/PMI.h>
 #include <GraphMol/Descriptors/DCLV.h>
 #include <GraphMol/Descriptors/BCUT.h>
+#ifdef RDK_BUILD_DESCRIPTORS3D
 #include <GraphMol/Descriptors/GETAWAY.h>
+#endif
 
 using namespace RDKit;
 
@@ -706,6 +708,7 @@ TEST_CASE(
   }
 }
 
+#ifdef RDK_BUILD_DESCRIPTORS3D
 TEST_CASE("Github #7264: GETAWAY descriptors are non-deterministic") {
   SECTION("as reported") {
     v2::SmilesParse::SmilesParserParams ps;
@@ -728,3 +731,4 @@ TEST_CASE("Github #7264: GETAWAY descriptors are non-deterministic") {
     }
   }
 }
+#endif

